flow/CMakeLists.txt

```diff
@@ -113,6 +113,9 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.cmake ${CMAKE_CURRENT_BINARY
 
 add_flow_target(STATIC_LIBRARY NAME flow SRCS ${FLOW_SRCS})
 target_link_libraries(flow PRIVATE stacktrace)
+if(NOT USE_ASAN AND NOT USE_VALGRIND AND NOT USE_TSAN AND NOT USE_UBSAN)
```

Comment:
We should probably add `NOT USE_MSAN` here as well

Reply:
```suggestion
if(NOT USE_ASAN AND NOT USE_VALGRIND AND NOT USE_TSAN AND NOT USE_UBSAN NOT USE_MSAN)
```
Good point.